### PR TITLE
Add canfarence DevfestTokyo

### PR DESCRIPTION
**Pull request by sgtmt**
DevfestTokyo2016

### CODE REVIEW

File: _conferences/devfest-tokyo-2016.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: "GDG DevFest Tokyo 2016"
```
Comment (JakeWharton) [inline _conferences/devfest-tokyo-2016.md]: Remove the year and country from the name

**Review by JakeWharton — CHANGES_REQUESTED** [review]

**Comment by hotchemi** [comment]
@sgtmt It's better to squash commits to one.

File: _conferences/devfest-tokyo-2016.md (hunk 1 above)
Comment (sgtmt) [reply _conferences/devfest-tokyo-2016.md]: And commit once again to remove the year and country

**Comment by sgtmt** [comment]
@hotchemi 

Thank you.
Once we organize and commit

**Pull request closed without merging**